docs/entities/rosetta/rosetta-construction-options.schema.json

```diff
@@ -87,6 +91,22 @@
     "signer_key": {
       "type": "string",
       "description": "The hex-encoded signer key (buff 33) for PoX."
+    },
+    "signer_private_key": {
+      "type": "string",
+      "description": "The hex-encoded signer private key for PoX. Must be specified for PoX transactions if `signer_signature` is not specified"
```

Comment:
`is not specified` is now outdated for these both

Reply:
Fixed